src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -106,37 +146,37 @@ PRPL (pronounced purple) is a set of concepts to enable the next generation of w
+This dependency chain can be slow to import so a compiler ([vulcanize](https://github.com/Polymer/vulcanize)) was written to combine all dependencies into one file so only one HTML import is needed.
 
-The PRPL pattern is the opposite of vulcanistion. This pattern only loads the minimum individual elements to show whats on screen, then lazily load the others.
+The PRPL pattern is the opposite of vulcanization. This pattern only loads the minimum individual elements to show whats on screen, then lazily load the others.
 
-Polymer's new [CLI](https://github.com/Polymer/polymer-cli) has as tooling for generating both the PRPL and vulcanised version of your project.
+Polymer's new [CLI](https://github.com/Polymer/polymer-cli) has as tooling for generating both the PRPL and vulcanized version of your project.
 
 ## Polybuild
 
-[Polybuild](https://github.com/PolymerLabs/polybuild) is a tool that lives on top of vulcanise and makes it super easy to vulcanise a file and split the output into two files: a HTML file for the [templates](http://www.html5rocks.com/en/tutorials/webcomponents/template/) and a JS file for the Polymer code.
+[Polybuild](https://github.com/PolymerLabs/polybuild) is a tool that lives on top of vulcanize and makes it super easy to vulcanize a file and split the output into two files: a HTML file for the [templates](http://www.html5rocks.com/en/tutorials/webcomponents/template/) and a JS file for the Polymer code.
 
-# Sanitising your components
+## Sanitising your components
 
-Polymer comes with a program called [Hydrolysis](https://github.com/Polymer/hydrolysis) that can be used to analyse elements. From this a tool called [polylint](https://github.com/PolymerLabs/polylint) was made that does a very basic sanity check of your elements (and follows the dependency chain) giving you some piece of mind in the build process.
+Polymer comes with a program called [Hydrolysis](https://github.com/Polymer/hydrolysis) that can be used to analyze elements. From this a tool called [polylint](https://github.com/PolymerLabs/polylint) was made that does a very basic sanity check of your elements (and follows the dependency chain) giving you some piece of mind in the build process.
 
 Polymer's [CLI](https://github.com/Polymer/polymer-cli) tool includes a linter to make the process easier.
 
-# Shady DOM
+## Shady DOM
 
-webcomponents.js comes in two flavours <em>standard</em> and <em>lite</em>. The <em>lite</em> version contains all the polyfills except the Shadow DOM polyfill. The reason for this is it is a heavy weight piece of code to copy the Shadow DOM!!
+webcomponents.js comes in two flavors <em>standard</em> and <em>lite</em>. The <em>lite</em> version contains all the polyfills except the Shadow DOM polyfill. The reason for this is it is a heavy weight piece of code to copy the Shadow DOM!!
```

Comment:
`The reason for this is it is a heavy weight piece of code to copy the Shadow DOM!!` to `The reason for this is that stubbing Shadow DOM is extremely inefficient and can impact performance of your application significantly!`